MapTopic: "/poly_map"

TargetTopic: "/move_base_simple/goal"

ZGoal: 1.0

MaxVel: 5.0

MaxAcc: 10.0

InitStep: 3.0   # initial step between near control points

PotentialWeight: 50000.0

SmoothWeight: 100.0

TOPPResolution: 10

TOPPPlotPath: "/home/star/Desktop/mr_ws/homeworks/finalProj_ws/pic"

# Map Parameters
MapBound: [-25.0, 25.0, -25.0, 25.0, 0.0, 5.0]

ObsVisHeight: 3.0

MinEdgeNum: 4

MaxEdgeNum: 7

ObsNum: 20

MaxDistCV: 5.0 # max distance between vertex of obstacle and center of obstacle

DilationRadius: 1.5

